server/webpa.go

```diff
@@ -257,6 +264,22 @@ func (w *WebPA) Prepare(logger log.Logger, health *health.Health, primaryHandler
 			ListenAndServe(logger, &w.Alternate, alternateServer)
 		}
 
+		if metricsServer := w.Metrics.New(logger, getMetricsHandler(gatherer)); metricsServer != nil {
+			infoLog.Log(logging.MessageKey(), "starting server", "name", w.Metrics.Name, "address", w.Metrics.Address)
+			ListenAndServe(logger, &w.Metrics, metricsServer)
+		}
+
 		return nil
 	})
 }
+
+//getMetricsHandler returns the handler for metrics given a gatherer. If none is provided, the default is used
+func getMetricsHandler(gatherer prometheus.Gatherer) http.Handler {
+	if gatherer == nil {
+		gatherer = prometheus.DefaultGatherer
+	}
+	//todo: need to add security layer decoration
+	mu := http.NewServeMux()
+	mu.Handle("/metrics", promhttp.HandlerFor(gatherer, promhttp.HandlerOpts{}))
```

Comment:
Add a HandlerOpts field to the WebPA struct, and pass that as a method parameter here as you did with the gatherer.  Don't silently assume a default options.